src/engine/framework/CommandSystem.cpp

```diff
@@ -193,12 +193,10 @@ namespace Cmd {
         commandLog.Debug("Execing command '%s'", command);
 
         std::string parsedString;
-        if (parseCvars) {
+        if (parseCvars)
             parsedString = SubstituteCvars(command);
-            command = parsedString;
-        }
 
-        Args args(command);
+        Args args(parseCvars? Str::StringRef(parsedString) : command);
```

Comment:
Didn't see this question. The cast is necessary because both operands of the ? operator need to resolve unambiguously.